training_record/tests/tests_views.py

```diff
@@ -0,0 +1,326 @@
+
+    def test_delete_recordcontent(self):
+        '''RecordContent should be deleted by DELETE request.'''
+        campus_event = mommy.make(training_event.models.CampusEvent)
+        record = mommy.make(training_record.models.Record,
+                            campus_event=campus_event)
+        recordcontent = mommy.make(training_record.models.RecordContent,
+                                   record=record)
+        url = reverse('recordcontent-detail', args=(recordcontent.pk,))
+
+        response = self.client.delete(url)
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(training_record.models.RecordContent.objects.count(),
+                         0)
+
+    def test_get_recordcontent(self):
+        '''RecordContent should be accessed by GET request.'''
+        campus_event = mommy.make(training_event.models.CampusEvent)
+        record = mommy.make(training_record.models.Record,
+                            campus_event=campus_event)
+        recordcontent = mommy.make(training_record.models.RecordContent,
+                                   record=record)
+        url = reverse('recordcontent-detail', args=(recordcontent.pk,))
+        expected_keys = {'id', 'create_time', 'update_time', 'record',
+                         'content_type', 'content'}
+
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(set(response.data.keys()), expected_keys)
+
+    def test_update_recordcontent(self):
+        '''RecordContent should be updated by PATCH request.'''
+        content_type0 = RecordContent.CONTENT_TYPE_CONTENT
+        content_type1 = RecordContent.CONTENT_TYPE_SUMMARY
+        campus_event = mommy.make(training_event.models.CampusEvent)
+        record = mommy.make(training_record.models.Record,
+                            campus_event=campus_event)
+        recordcontent = mommy.make(training_record.models.RecordContent,
+                                   record=record, content_type=content_type0)
+        url = reverse('recordcontent-detail', args=(recordcontent.pk,))
+        data = {'content_type': content_type1}
+
+        response = self.client.patch(url, data, format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn('content_type', response.data)
+        self.assertEqual(response.data['content_type'], content_type1)
+
+
+class TestRecordAttachmentViewSet(APITestCase):
+    '''Unit tests for RecordAttachment view.'''
+    def test_create_recordattachment(self):
+        '''RecordAttachment should be created by POST request.'''
+        url = reverse('recordattachment-list')
+        campus_event = mommy.make(training_event.models.CampusEvent)
+        record = mommy.make(training_record.models.Record,
+                            campus_event=campus_event)
+        attachment_type = 0
```

Comment:
Do not use 0 directly, you can use `RecordAttachment.ATTACHMENT_TYPE_CONTENT` to replace it.